webpack.config.js

```diff
@@ -20,16 +28,27 @@ module.exports = {
             path.resolve(__dirname, 'src/bower_components/apiclient'),
             path.resolve(__dirname, 'src/bower_components/apiclient/sync'),
             path.resolve(__dirname, 'src/components/cardbuilder'),
```

Comment:
Do we still need the above modules or can we remove them?

Reply:
They have been removed now.